_posts/blog/2022-06-22-2.1.0-metrics-and-tracing.md

```diff
@@ -149,3 +149,24 @@ docker run -d --rm --name jaeger \
 8. View traces in Jaeger UI at http://localhost:16686. You can select the service name on the left panel and click `Find Traces` to view the trace information. If everything is working correctly, then you should see something like the image below.
 
 ![Jaeger Screenshot](/images/blog/202206_metrics_and_tracing/Jaeger_Screenshot.png)
+
```

Comment:
The blog post was for the 2.1.0 release. It looks like the addition here updates / replaces items 5 and 6 above. I'm thinking that we should put a note in items 5 and 6 that says something like:

```
This contains instructions for configuring OpenTelemetry and Jaeger with Accumulo 2.1.0. As of Accumulo 2.1.4 the version of OpenTelemetry has been updated and the instructions have changed. See below for more information.
```

Reply:
Good idea. Added in 4b88f58.

Reply:
![Screenshot from 2025-04-08 14-52-32](https://github.com/user-attachments/assets/7959cd05-21aa-4816-af91-9140f566e8a0)
![Screenshot from 2025-04-08 14-52-18](https://github.com/user-attachments/assets/858d23c1-c2de-437b-88f8-763267f14297)